docker-compose.yml

```diff
@@ -16,6 +16,19 @@ services:
       - PGDATA=/var/lib/postgresql/data/pgdata
     expose:
       - "5432"
+
+  certs:
+    image: registry.access.redhat.com/ubi8/ubi:latest
+    restart: unless-stopped
+    command: sh -c "sh /etc/pki/ca-trust/source/anchors/dodcerts.sh && update-ca-trust && tail -f /dev/null"
```

Comment:
add logging between all the phases

Reply:
Added a couple echos for start/end of script + if the internet check fails.  Otherwise, I enabled xtrace so it'll by default log every command taken.